addons/binding/org.openhab.binding.evohome/README.md

```diff
@@ -0,0 +1,47 @@
+# evohome Binding
+
+This binding connects your to your Honeywell evohome system, similar to what the offical app does. This means that an internet connection is needed. Currently it is possible to set the global system mode and read the values of setpoints and temperatures of individual zones.
+
+
+
+## Supported Things
+
+The binding supports the following things:
+
+* evohome Account
```

Comment:
I actually quite deliberately spelled it as 'evohome' since the product name is stylized like that. It it customary within openhab to spell product names with a capital?

Reply:
You are so right, if you google honeywell's site spells it with a capital but if you click you see the real thing, all lowercase :-)